.github/renovate.json

```diff
@@ -10,43 +10,19 @@
   ],
   "packageRules": [
     {
-      "matchManagers": ["npm"],
+      "matchManagers": ["npm", "nuget"],
```

Comment:
@jattasNI do youu think this missing is why renavate ended up making three separate .net related PRs? That was my theory

- https://github.com/ni/nimble/pull/1887
- https://github.com/ni/nimble/pull/1884
- https://github.com/ni/nimble/pull/1881

Reply:
Yeah, partly. We enabled the `nuget` manager above and then left it with largely its default configuration. The default configuration includes [Renovate presets](https://docs.renovatebot.com/key-concepts/presets/) where they purposely batch [groups of related dependencies](https://docs.renovatebot.com/presets-monorepo/) together. The only change we made to that for nuget was to also group the bunit dependencies together since they didn't have a preset for that.

For the `npm` manager we took the opposite approach and disabled all the defaults here. That turns off all the presets and then we explicitly opted back in to our own configuration of groups.

I believe your current configuration will achieve the goal of one PR for all npm dependencies and devDependencies and another PR for all nuget dependencies. I'm less sure whether that's a good idea (Renovate's docs really encourage using their default settings) but I'm willing to try it out and see if we like it.